apps/journeys-admin/src/components/TemplateCustomization/MultiStepForm/MultiStepForm.tsx

```diff
@@ -122,11 +122,12 @@ export function MultiStepForm(): ReactElement {
             {t('Edit Manually')}
           </Button>
         </NextLink>
-
-        <ProgressStepper
-          activeStepNumber={screens.indexOf(activeScreen)}
-          totalSteps={totalSteps}
-        />
+        {totalSteps > 3 && (
```

Comment:
use screens assertion rather than total steps. We could add/remove screens in the future causing this to fail so assert the render logic by screens. you can get the `hasEditableText` and `hasCustomizableLinks` variables from `getCustomizeFlowConfig` function


```suggestion
        {hasEditableText && hasCustomizableLinks && (
```